docs/CONNECTOR_PROTOCOL.md

```diff
@@ -96,8 +96,8 @@ This is our main communication index, used to communicate the connector's config
                            regularly (UTC)
   last_sync_error: string;   -> Optional last sync error message
   last_synced: date;    -> Date/time of last completed sync (UTC)
-  last_indexed_count: number;    -> How many documents were inserted into the index
-  last_deleted_count: number;    -> How many documents were deleted from the index
+  last_indexed_document_count: number;    -> How many documents were inserted into the index
```

Comment:
I renamed the field name as I suggested [here](https://github.com/elastic/connectors-ruby/pull/282#discussion_r959697636), to be more consistent.

cc @sphilipse in case this affects FE

Reply:
Drafted a separate PR https://github.com/elastic/connectors-ruby/pull/287